### PR TITLE
[no ticket][risk=no] Tweaking e2e tests to reduce runtime

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

File: e2e/app/component/workspace-card.ts [hunk 1]
```diff
@@ -43,13 +43,12 @@ export default class WorkspaceCard extends CardBase {
    * @throws TimeoutError if fails to find Card.
    */
   static async findAllCards(page: Page, accessLevel?: WorkspaceAccessLevel): Promise<WorkspaceCard[]> {
-    try {
-      await waitWhileLoading(page);
-      await page.waitForXPath(WorkspaceCardSelector.cardRootXpath, { visible: true, timeout: 1000 });
-    } catch (e) {
-      return [];
-    }
```
Comment (aweng98) [inline e2e/app/component/workspace-card.ts:51]: Code is problematic. Say if `waitWhileLoading` has failed, an empty array is returned by function.

File: e2e/app/component/workspace-card.ts [hunk 2]
```diff
@@ -43,13 +43,12 @@ export default class WorkspaceCard extends CardBase {
...
-    const workspaceCards = (await page.$x(WorkspaceCardSelector.cardRootXpath)).map((card) =>
+    await waitWhileLoading(page);
+    await page.waitForXPath(WorkspaceCardSelector.cardRootXpath, { visible: true, timeout: 10000 }).catch(() => {
```
Comment (chavanr2019) [inline e2e/app/component/workspace-card.ts:47]: @aweng98 overall LGTM! Whenever possible could you please explain the `.catch(()` , just want to have a better understanding. Thanks!

**Review by chavanr2019 — APPROVED** [review]
LGTM!

File: e2e/app/component/workspace-card.ts (hunk 2 above)
Comment (aweng98) [reply e2e/app/component/workspace-card.ts:47]: I think it's easier to read. [Here](https://stackoverflow.com/questions/58908381/javascript-try-catch-vs-catch-chain) is good explanation.

**Review by aweng98 — COMMENTED** [review]
thank you

**Pull request merged**